creating_images/guidelines.adoc

```diff
@@ -299,45 +299,34 @@ privileged user.
 Because the user ID of the container is generated dynamically, it will not have
 an associated entry in *_/etc/passwd_*. This can cause problems for applications
 that expect to be able to look up their user ID. One way to address this problem
-is to use link:https://cwrap.org/nss_wrapper.html[nss wrapper] and dynamically
-create a *_passwd_* file with the container's user ID as part of the image's
-start script:
+is to dynamically create a *_passwd_* file entry with the container's user ID as part
+of the image's start script. This is what a Dockerfile might include:
 
 ----
-export USER_ID=$(id -u)
-export GROUP_ID=$(id -g)
-envsubst < ${HOME}/passwd.template > /tmp/passwd
-export LD_PRELOAD=/usr/lib64/libnss_wrapper.so
-export NSS_WRAPPER_PASSWD=/tmp/passwd
-export NSS_WRAPPER_GROUP=/etc/group
+RUN chgrp -R 0 /some/directory \
+    && chmod -R g=u /some/directory /etc/passwd
```

Comment:
Agree w/ '&&' on prior line... had just kept how it was in prior docs, will change.

In regards to the -R, that is required for '/some/directory'. Begs the question though, do we repeat the /some/directory `chgrp` and `chmod` since its also shown above? or do we limit this docs section to the single file `chmod g=u /etc/passwd`?  Folks can then click on one of the links below for a full working Dockerfile as a reference for piecing it all together.